design-at-cornell/src/events/Events.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { PageContainer, EventList } from './EventsStyles';
+import Title from './Title';
+import Filter from './dashboard/Filter';
+import EventDisplay from './dashboard/EventDisplay';
+
+export type Event = {
+  name: string;
+  time: string;
+  location: string;
+  tags: string[];
+  description: string;
+  links: string[];
+};
+
+const Events = () => {
+  const [events] = useState<Event[]>([
+    {
+      name: 'Design Seminar Series with Professor XYZ',
+      time: '3:00 - 4:00 PM EST Saturday, May 1, 2021',
+      location: 'Virtual',
+      tags: ['Annual', 'Virtual', 'Computing in the Arts'],
```

Comment:
Since these both have location in them, would it be possible to include the location in the tags instead of stating it twice here?

Reply:
I think the tags are just placeholders for now (I added the same tags in the Figma design). We might still want to decide what tags to include, which might not include the location at all since there is already a dedicated place to display the location.